dlp/snippets/deid.py

```diff
@@ -1023,6 +1023,123 @@ def deidentify_with_exception_list(
 # [END dlp_deidentify_exception_list]
 
 
+# [START dlp_deidentify_table_row_suppress]
+def deidentify_table_suppress_row(
+    project,
+    table_data,
+    condition_field,
+    condition_operator,
+    condition_value
+):
+    """ Uses the Data Loss Prevention API to de-identify sensitive data in a
+      table by suppressing entire row/s based on a condition.
+
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        table_data: Json string representing table data.
+        condition_field: A table Field within the record this condition is evaluated against.
+        condition_operator: Operator used to compare the field or infoType to the value. One of:
+            RELATIONAL_OPERATOR_UNSPECIFIED, EQUAL_TO, NOT_EQUAL_TO, GREATER_THAN, LESS_THAN, GREATER_THAN_OR_EQUALS,
+            LESS_THAN_OR_EQUALS, EXISTS.
+        condition_value: Value to compare against. [Mandatory, except for ``EXISTS`` tests.].
+
+    Returns:
+        De-identified table is returned;
+        the response from the API is also printed to the terminal.
```

Comment:
Capitalize "the"

Reply:
added type hints, removed return table as mentioned in one of the PRs, "Returns" in the doc string is removed as it returns none now